Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py

```diff
@@ -225,6 +227,49 @@ def get_paged_search_secrule(self, auth_token: str, payload: Dict[str, Any]):
         )
         return secrule_page_search_response
 
+    def get_paged_all_collectors(self, auth_token: str, payload: Dict[str, Any]):
+        """Calling get paged search api for collector
+
+        Args:
+            auth_token (str): authentication token
+            payload (Dict[str, Any]): payload to be used for making request
+        """
+        parameters: Dict[str, Any] = {
+            "pageSize": payload["pageSize"],
+            "page": payload["page"],
+        }
+
+        paged_all_collectors_response = self._http_request(
+            method="GET",
+            url_suffix=COLLECTOR_URL,
+            params=parameters,
+            headers={
+                "Content-Type": "application/json",
+                "Accept": "application/json",
+                "X-FM-Auth-Token": auth_token,
+            },
+        )
+        return paged_all_collectors_response
+
+    def get_collector_status_byid(self, auth_token: str, payload: Dict[str, Any]):
```

Comment:
```suggestion
    def get_collector_status_byid(self, auth_token: str, collector_id: int):
```
better to pass the collector_id straight to the methods instead of using payload